nautobot/utilities/utils.py

```diff
@@ -382,37 +422,86 @@ def copy_safe_request(request):
     )
 
 
-def get_filterset_for_model(model):
-    """Return the FilterSet class associated with a given model.
+def get_object_for_model(model, module_name, object_suffix):
```

Comment:
Perhaps `get_related_class_for_model` or something would be more obvious than `get_object_for_model`?